src/types/FilterEnum.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
+  all = 'All',
+  active = 'active',
+  completed = 'completed',
```

Comment:
```suggestion
  active = 'Аctive',
  completed = 'Сompleted',
```

Reply:
I can't fix this code because knocks out the error